docs/tutorial/custom_frontend.rst

```diff
@@ -0,0 +1,323 @@
+.. _custom-apps:
+
+Creating a custom Checkbox frontend
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This guide describes how to create a custom Checkbox frontend for testing a
+new project. This is usually done to package in-development tests and test
+plans before shifting them to the main Checkbox providers. This tutorial will
+guide you to package as a snap the tutorial provider we have previously
```

Comment:
```suggestion
guide you to package as a snap the :ref:`tutorial provider <adv_test_case>` we have previously
```
I assume you are referring to the `2024.com.tutorial:tutorial` example one, not the pre-defined `certification:tutorial`, linking back to that page would be helpful